src/settings/toml/target.rs

```diff
@@ -33,6 +80,10 @@ impl Target {
         self.kv_namespaces = Some(updated_namespaces);
     }
 
+    pub fn remove_all_kv_namespaces(&mut self) {
```

Comment:
this refactor is a great improvement overall, but this method sort of gives me pause. i don't have a better answer though so seems good for now.

Reply:
it's used for unauthenticated preview and nowhere else - agree it's a little weird